src/org/usfirst/frc/team2175/properties/WiringProperties.java

```diff
@@ -8,26 +8,21 @@
     private int leftSlaveMotorTwoNumber;
     private int rightSlaveMotorOneNumber;
     private int rightSlaveMotorTwoNumber;
-
-    private int driveShiftersForwardNumber;
-    private int driveShiftersReverseNumber;
+    private String[] driveShiftersSolenoidInfo;
```

Comment:
@4thwind Again, let's make a little `SolenoidInfo` class instead of storing everything as a string array.